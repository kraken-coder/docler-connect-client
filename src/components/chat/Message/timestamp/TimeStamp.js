import React from 'react'
import PropTypes from 'prop-types'

import { time12Hours, time24hours } from '../../../../utils/general'

const TimeStamp = ({ clockMode }) => (
  <div className="msg-bottom">
    <p className="msg-time">{clockMode === '24' ? time12Hours() : time24hours()}</p>
  </div>
)

TimeStamp.propTypes = {
  clockMode: PropTypes.string.isRequired,
}

export default TimeStamp
