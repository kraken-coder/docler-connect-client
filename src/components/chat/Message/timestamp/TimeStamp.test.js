import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import TimeStamp from './TimeStamp'

test('Renders Without Crashing', () => {
  const Wrapper = shallow(<TimeStamp />)
  expect(toJSON(Wrapper)).toMatchSnapshot()
})
