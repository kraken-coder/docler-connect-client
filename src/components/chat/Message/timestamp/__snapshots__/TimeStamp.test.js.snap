// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Without Crashing 1`] = `
<div
  className="msg-bottom"
>
  <p
    className="msg-time"
  >
    10:08
  </p>
</div>
`;
